cpuset/cpuset_test.go

```diff
@@ -0,0 +1,332 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cpuset
```

Comment:
Worth to add comment on where this file got copied over?

Reply:
Going to defer this change to k/utils, since it doesn't apply until it hits k/utils

Reply:
Yes, @SergeyKanzhelev exactly. To clarify, I'm going to push a PR to k/k with most of these changes, except these two 'origin comment' ones. Was just acknowledging the feedback.

Reply:
Added an origin comment to cpuset.go only